modules/arm_plugin/src/arm_converter/arm_converter_sequence_ops.cpp

```diff
@@ -0,0 +1,271 @@
+                                                    reinterpret_cast<const char*>(H),
+                                                    H_shape,
+                                                    reinterpret_cast<const char*>(C),
+                                                    C_shape,
+                                                    reinterpret_cast<const char*>(seq_lengths),
+                                                    seq_lengths_shape,
+                                                    reinterpret_cast<const char*>(W),
+                                                    W_shape,
+                                                    reinterpret_cast<const char*>(R),
+                                                    R_shape,
+                                                    reinterpret_cast<const char*>(B),
+                                                    B_shape,
+                                                    reinterpret_cast<char*>(Y),
+                                                    reinterpret_cast<char*>(Ho),
+                                                    reinterpret_cast<char*>(Co),
+                                                    activation_f,
+                                                    activation_g,
+                                                    activation_h,
+                                                    clip,
+                                                    direction);
+}
+
+
+template<> Converter::Conversion::Ptr Converter::Convert(const opset::LSTMSequence& node) {
+    auto make = [&] (auto refFunction) {
+    return this->MakeConversion(refFunction,
+                                node.input(0),
+                                node.get_input_shape(0),
+                                node.input(1),
+                                node.get_input_shape(1),
+                                node.input(2),
+                                node.get_input_shape(2),
+                                node.input(3),
+                                node.get_input_shape(3),
+                                node.input(4),
+                                node.get_input_shape(4),
+                                node.input(5),
+                                node.get_input_shape(5),
+                                node.input(6),
+                                node.get_input_shape(6),
+                                node.output(0),
+                                node.output(1),
+                                node.output(2),
+                                node.get_activations()[0],
+                                node.get_activations()[1],
+                                node.get_activations()[2],
+                                node.get_clip(),
+                                node.get_direction());
+    };
+    SWITCH_TYPES(node.get_input_element_type(0), node.get_input_element_type(3), wrap_lstm_sequence, make)
```

Comment:
Why we cannot use 
```cpp
    return CallSwitch(
        AP_WRAP(make, wrap_lstm_sequence),
        node.input(3), intTypes,
        node.input(0), floatTypes);
```
instead of `SWITCH_TYPES`?

Reply:
done, remove `SWITCH_TYPES` and use yours way